reuse/contribute-to-docs.txt

```diff
@@ -2,21 +2,21 @@
 
 Start: How to contribute
 
-These docs are on located on a GitHub repository at: `ubuntu-cloud-docs`_ and you'll need a GitHub account to make contributions. It is a good idea to fork this repository into your own account before you start, otherwise GitHub will anyway prompt you to do so when you attempt your first change.
+These docs are located on the GitHub repository named `ubuntu-cloud-docs`_, and you'll need a GitHub account to make contributions. It is a good idea to fork this repository into your account before you start, otherwise, GitHub will prompt you to do so when you attempt your first change. 
 
-The docs are:
+The documentations are:
```

Comment:
Something seems off here, it probably sounds a bit too formal. Maybe instead we could use "This documentation set is:"?